code/go/0chain.net/smartcontract/dbs/event/process.go

```diff
@@ -68,6 +69,13 @@ func (edb *EventDb) addStat(event Event) error {
 		return edb.updateBlobber(updates)
 	case TagDeleteBlobber:
 		return edb.deleteBlobber(event.Data)
+	case TagAddTransaction:
+		var transaction Transaction
+		err := json.Unmarshal([]byte(event.Data), &transaction)
+		if err != nil {
+			return err
+		}
+		return edb.AddTransaction(transaction)
```

Comment:
This should be called from the above block where I have commented which shall add the transaction in the db